packages/sync/Module_Full_Sync.php

```diff
@@ -389,7 +391,7 @@ public function clear_status() {
 
 		$this->delete_enqueue_status();
 
-		foreach ( Jetpack_Sync_Modules::get_modules() as $module ) {
+		foreach ( Modules::get_modules() as $module ) {
 			Jetpack_Options::delete_raw_option( "{$prefix}_{$module->name()}_sent" );
```

Comment:
should we have a `Jetpack_Options` package? @zinigor and @gititon are also using these in  https://github.com/Automattic/jetpack/pull/12399 and https://github.com/Automattic/jetpack/pull/12516/

how do you all imagine sharing these?

Reply:
Yeah, I think we should take care of those first, and then start on the sync one.